datatypes/src/commonMain/kotlin/at/asitplus/crypto/datatypes/CryptoPublicKey.kt

```diff
@@ -71,11 +71,11 @@ sealed class CryptoPublicKey : Asn1Encodable<Asn1Sequence>, Identifiable {
             val keyBytes = decoded.copyOfRange(2, decoded.size)
 
             return when (codec) {
-                0x1200uL, 0x1290uL ->
```

Comment:
Why don't we still support our inofficial values?

Reply:
I was under the impression this was sorted out with @iaik-jheher

Reply:
Encoding just the "official" values is fine, but why do we need to drop support for decoding it?